src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
+import { filterByTitle, filterByTodoStatus } from './helpers';
 import { Loader } from './components/Loader';
+import { LoadError } from './types/ErrorType';
+import { LoadingError } from './components/LoadingError/LoadingError';
+import { TodosFilterType } from './types/TodosFilterType';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [showTodoModal, setShowTodoModal] = useState(false);
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [filterBy, setFilterBy] = useState(TodosFilterType.Default);
+  const [query, setQuery] = useState('');
+  const [isLoading, isSetLoading] = useState(false);
+  const [isLoadingError, isSetLoadingError] = useState(false);
```

Comment:
you misspelled setState name function 
isSetLoadingError -> setIsLoadingError
you can name it shorter, for example hasError, but it's up to you